conductr/tasks/main.yml

```diff
@@ -43,22 +38,23 @@
   become_method: sudo
   when: "{{ ENABLE_DEBUG }} == true"
 
-- shell: 'echo -Dconductr-haproxy.ip=$(hostname -i) | sudo tee -a /usr/share/conductr-haproxy/conf/application.ini'
+- shell: 'echo -Dakka.cluster.roles.{{ item.0 }}={{ item.1 }} | sudo tee -a /usr/share/conductr/conf/application.ini'
   become: yes
   become_method: sudo
+  with_indexed_items: "{{ CONDUCTR_ROLES }}"
```

Comment:
@edwardcallahan - the `CONDUCTR_ROLES` variable may be loaded from vars file, else fallback to default roles.